### ISSUE TITLE
Close Account

### ISSUE THREAD

**Issue by cbj4074** [indiehd/web-api#26]

**Comment by poppabear8883** [comment]
This should be in progress now with the Observers ?

**Comment by cbj4074** [comment]
Yes, ultimately, this should be implemented through the Observer pattern, as we've done in similar scenarios, but we should flesh-out the "complex validation rules" that I noted previously. I'll add another comment shortly with more information.

**Comment by cbj4074** [comment]
We should use the existing documentation as a starting-point; currently, this is how the process works:

https://indiehd.com/docs/terms-of-use/#9c

Provision 3 no longer applies, and can be removed. Also, we should consider revising this to be GDPR-compliant.
